src/components/AppPaginatedResult.vue

```diff
@@ -45,6 +70,11 @@ const currentPageSize = computed({
   set: (newPageSize) => emit('update:pageSize', newPageSize),
 });
 
+const pageSizes = [12, 25, 50, 100].map((x) => ({
+  id: x,
+  label: x + ' per page',
```

Comment:
Needs internationalising. Note that translation is "reactive", meaning it should be wrapped in a `computed` so it updates if the locale changes, e.g. something like (not tested)

```ts
const pageSizes = computed(() => [12, 25, 50, 100].map(x => ({
  id: x,
 label: t('the.key.here', {page: n(x)})
}))
```

Note that the page number is passed as a `page` property and is i18ned itself using the `n` i18n function which you can import with
```ts
const { t, n } = useI18n();
```
This doesn't make any difference for small numbers but 1000 would become "1,000" (in EN), and it's always good practice to i18n numbers in case there is a language out there than does something with "100"